### ISSUE TITLE
Types\SellerProfilesType -- ProfileID Int larger than 32bit php_int_max

### ISSUE THREAD

**Issue by Rykono** [davidtsadler/ebay-sdk-php#191]

**Comment by michabbb** [comment]
[nope, you have to switch to 64bit](http://devbay.net/sdk/guides/getting-started/requirements.html#using-the-sdk-with-32-bit-systems)

**Comment by Rykono** [comment]
Just changed int to string works fine. Here for anyone else that runs into it.

**Comment by michabbb** [comment]
that´s a bad hack. next time this SDK gets an update, you may not forget to redo all your hacks. use 64bit, we no longer live in the stone age.

**Comment by Rykono** [comment]
what does bad hack mean?

**Comment by michabbb** [comment]
it´s the opposite of something good. i guess you can close this issue because there is nothing in here that helps somebody.

**Comment by Rykono** [comment]
If someone is running this on a 32 bit system and ran into this problem, would this not help them?

**Comment by michabbb** [comment]
no. this person should switch to 64bit - that´s the only solution that makes sense when working with this SDK and ebay at all.

btw. this issue is already adressed [here](https://github.com/davidtsadler/ebay-sdk-php) and more detailed [here](http://devbay.net/sdk/guides/getting-started/requirements.html#using-the-sdk-with-32-bit-systems).

**Comment by Rykono** [comment]
Whats the point of the integer data type requirement? Since xml doesn't send data types, ebay would kick back an error anyway, if the input was wrong. Im no php api expert, so your knowledge on the subject would be appreciative.

**Comment by michabbb** [comment]
Pardon, my intention is to help, not to discuss why it makes sense to use a 32bit system these days, nothing more this is about. I'm out.

**Comment by Rykono** [comment]
I guess you misunderstood the question?

**Comment by medigeek** [comment]
Duplicate of #101 - just to keep track of things. :)
